packages/entities/CHANGELOG.md

```diff
@@ -2,6 +2,15 @@
 
```

Comment:
I'd suggest creating a new PR with only the docs changes and choosing the right commit message.

Reply:
😅,
Indeed it make sense to update only the docs
I'll do it